openhcl/openhcl_boot/src/dt.rs

```diff
@@ -158,6 +160,7 @@ pub fn write_dt(
     cmdline: &ArrayString<COMMAND_LINE_SIZE>,
     sidecar: Option<&SidecarConfig<'_>>,
     boot_times: Option<BootTimes>,
+    #[allow(unused_variables)] isolation_type: IsolationType, // isolation_type is unused on aarch64
```

Comment:
replace `allow` with `expect` which requires justification, and expects the lint will fire.